### ISSUE TITLE
`objed-ace` ignores nested objects

### ISSUE THREAD

**Issue by WhatisRT** [clemera/objed#43]
I'm not sure if this always happens, but it happens with the `bracket` object. If I have a `bracket` selected, then invoking `objed-ace` only cares about brackets at the top level. In the following example, there is no way to jump to `(bar)` with `objed-ace`:
```
(foo (bar))
(test)
```

**Comment by clemera** [comment]
That's right, this is the case for any nested object type because any internal nodes are skipped. This should be fixed together with  #39.
